embedded-graphics/src/lib.rs

```diff
@@ -81,23 +91,27 @@
 //! cargo run -p embedded-graphics-simulator --example hello
 //! ```
 //!
-//! [simulator]: https://github.com/jamwaffles/embedded-graphics/tree/c4f74c12dae9f0a0193fa48192f905a002bf8c9d/simulator
-//! [simulator examples]: https://github.com/jamwaffles/embedded-graphics/tree/c4f74c12dae9f0a0193fa48192f905a002bf8c9d/simulator/examples
+//! [simulator]: https://github.com/jamwaffles/embedded-graphics/tree/master/simulator
+//! [simulator examples]: https://github.com/jamwaffles/embedded-graphics/tree/master/simulator/examples
 //!
 //! # Crate features
 //!
-//! Add these to your `Cargo.toml` to turn on extra bits of functionality.
+//! Additional features can be enabled by adding the following features to your `Cargo.toml`.
 //!
 //! * `nalgebra_support` - use the [Nalgebra](https://crates.io/crates/nalgebra) crate with `no_std`
 //! support to enable conversions from `nalgebra::Vector2` to [`Point`] and [`Size`].
 //!
 //! * `fixed_point` - use fixed point arithmetic instead of floating point for all trigonometric
 //! calculation.
 //!
-//! # Implementing `embedded_graphics` in a driver
+//! # Migrating from 0.5 to 0.6
+//!
+//! Please read [the migration guide](https://github.com/jamwaffles/embedded-graphics/blob/master/embedded-graphics/MIGRATING-0.5-0.6.md).
 //!
-//! To add support for embedded_graphics to a display driver, [`DrawTarget`] should be implemented.
-//! This allows all embedded_graphics objects to be rendered by the display. See the [`DrawTarget`]
+//! # Implementing `embedded_graphics` supprot for a driver
+//!
+//! To add support for embedded-graphics to a display driver, [`DrawTarget`] should be implemented.
+//! This allows all embedded-graphics objects to be rendered by the display. See the [`DrawTarget`]
```

Comment:
Should we use a consistent way to refer to "embedded-graphics objects" in the docs? The list on the top of the readme uses "items" instead. Or we could use "drawables", because of the `Drawable` trait.

Reply:
I've kind of standardised on `items`, but I'd prefer not to go through all the readmes/docs in this PR to standardise that wording. I've changed this case in particular from `objects` to `items` though.